pkg/os/exec.go

```diff
@@ -16,6 +19,7 @@ func RunWithPrivilege(cmdAndArgs ...string) (string, string, error) {
 	stdErr := new(bytes.Buffer)
 	cmd.Stdout = stdOut
 	cmd.Stderr = stdErr
+	logging.Infof("Will use root access to %s\n", reason)
```

Comment:
I'd prefer that we avoid creating full sentences like this, this is going to be error prone if one does not pay attention how this works. "Will use root access: %s" would work better I think.